### DISCUSSION TITLE
Server Actions don't work on vercel

### DISCUSSION THREAD

**Discussion by Arinji2** [vercel/community#4520] [Bug Report]
Hilo, so I just get this error on vercel (which i dont get locally)
```
Error: Cannot find module 'next/dist/compiled/next-server/app-page-experimental.runtime.prod.js'
Require stack:
```
https://github.com/Arinji2/vibeify
Any ideas?

Tried updating and it works on local prod.. just not on vercel

**Comment by leerob** [comment]
Duplicate of https://www.reddit.com/r/nextjs/comments/17ak6bv/server_actions_dont_work_on_vercel/, we can continue there.
